sklearn/externals/funcsigs.py

```diff
@@ -511,7 +511,7 @@ def __init__(self, parameters=None, return_annotation=_empty,
 
     @classmethod
     def from_function(cls, func):
-        '''Constructs Signature for the given python function'''
```

Comment:
Please do not edit files under `sklearn/externals`. Changes should happen only in upstream projects.